.github/workflows/package.yaml

```
@@ -0,0 +1,29 @@
+
+name: Python package
```

This workflow tends to be executed on push or PR; therefore the workflow name `Python package` and the file name `package` does not fit the purpose. Changing it to `ci` or anything related would be more comprehensible.

```suggestion
name: CI
```